.circleci/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+cd /tmp/pytorch/xla
+
+export XLA_USE_XRT=1 XRT_DEVICE_MAP="CPU:0;/job:localservice/replica:0/task:0/device:XLA_CPU:0"
+export XRT_WORKERS="localservice:0;grpc://localhost:40934"
+
+python test/test_operations.py
+python test/test_train_mnist.py --tidy
+
+pushd test/cpp
+export CC=clang-7 CXX=clang++-7
+./run_tests.sh
```

Comment:
This will need the same `export CC=clang-7 CXX=clang++-7` as the build script, this appears to be the last problem we have to solve. Is there a way to set these environment variables in one place?

Reply:
Sure, we can hack this into a separate `xla_env`. This cannot go into `env` as we will have to use it for pytorch build. Let me fix it.